CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+### Added
+ - Literate can now output [Quarto](https://quarto.org/) notebooks (markdown documents with
+   the `.qmd` file extension) by passing `flavor = Literate.QuartoFlavor()` to
+   `Literate.markdown`. This feature is marked as experimental since it has not been widely
+   tested and the Quarto-specific syntax may change before Literate version 3 depending on
+   what the community wants or needs. ([#199][github-199], [#200][github-200])
```

Comment:
I marked this as experimental for now so that we (you) have the liberty to make changes to the specific format.